### PR TITLE
Added params for VerifyHostKeyDNS in ssh_config

### PR DESCRIPTION
**Pull request by kengribble**
It's my first pull request and I'm fairly new to puppet, so please forgive any missteps!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,6 +147,13 @@ Array of ciphers to be used with the MACs option in ssh_config.
 
 - *Default*: undef
 
+ssh_config_verify_host_key_dns
+---------------
+VerifyHostKeyDNS: Enables/disables VerifyHostKeyDNS DNSSEC validation in ssh_config. Valid values are 'yes' and 'no'.
```
Comment (ghoneycutt) [inline README.md]: 'ask' is also valid as per https://www.freebsd.org/cgi/man.cgi?ssh_config(5)

File: manifests/init.pp [hunk 2]
```diff
@@ -28,6 +28,7 @@
   $ssh_config_macs                      = undef,
   $ssh_config_use_roaming               = 'USE_DEFAULTS',
   $ssh_config_template                  = 'ssh/ssh_config.erb',
+  $ssh_config_verify_host_key_dns       = undef,
```
Comment (ghoneycutt) [inline manifests/init.pp]: Each parameter must be validated. The validation for this is

```puppet
if $ssh_config_verify_host_key_dns != undef {
  validate_re($ssh_config_verify_host_key_dns, /^(yes|no|ask)$/, ssh::ssh_config_verify_host_key_dns may be 'yes', 'no' or 'ask' and is set to <${ssh_config_verify_host_key_dns}>.")
}
```

File: templates/ssh_config.erb [hunk 3]
```diff
@@ -109,3 +109,6 @@ GSSAPIDelegateCredentials <%= @ssh_gssapidelegatecredentials %>
 <% if @ssh_config_user_known_hosts_file -%>
   UserKnownHostsFile <%= @ssh_config_user_known_hosts_file.join(' ') %>
 <% end -%>
+<% if @ssh_config_verify_host_key_dns != nil -%>
```
Comment (ghoneycutt) [inline templates/ssh_config.erb]: This needs spec tests. They will be similar to https://github.com/ghoneycutt/puppet-module-ssh/commit/7c96719526b5c1fe9bbd1d1213cd766a71ee6c92

**Comment by ghoneycutt** [comment]
Thanks for the contribution @kengribble !

I've made a few comments for changes. Let me know if you need help with the spec tests and I can add them. Thought you might want to take a try at them though.

**Comment by Phil-Friderici** [comment]
Closing this PR as it was made for the v3 version of the module which is outdated now.
v4 does allow to set `UserKnownHostsFile` in ssh_config with the `$ssh::user_known_hosts_file` parameter.

**Pull request closed without merging**